tests/migration/migration.go

```diff
@@ -752,6 +754,56 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 			})
 		})
 
+		Context("[Serial] already running", Serial, func() {
+			var vmi *v1.VirtualMachineInstance
+
+			BeforeEach(func() {
+				tests.EnableDownwardMetrics(virtClient)
```

Comment:
Same as above, it looks strange to me that we call this in BeforeEach/AfterEach and in the global test setup.